modal.js

```diff
@@ -54,69 +52,63 @@ const data = [
+for (let i = 0; i < data.length; i += 1) {
+  projectsList.innerHTML = `${projectsList.innerHTML}
+   <div class="content">
+            <div class="snapshoot">
+              <img
+                class="snapshoot1-mob"
+                src="${data[i].image}"
+                alt="snapshoot1"
+              />
+              <img
+                class="snapshoot1-desk"
+                src="${data[i].image}"
+                alt="snapshoot1"
+              />
             </div>
-            <div class="modal-mobile-detail">
-              <!-- <div> -->
-                <p class="modal-description desk mob"></p>
-              <div class="tech">
-                <ul class="modal-tools mobil">
-                  <li class="tool">${elem.html}</li>
-                  <li class="tool">${elem.css}</li>
-                  <li class="tool">${elem.javascript}</li>
-                </ul>
-                <ul class="modal-tools desk">
-                  <li class="tool">github</li>
-                  <li class="tool">ruby</li>
-                  <li class="tool">Bootstrap</li>
-                </ul>
-                <button type="button" class="modal-btn">
-                  See Live <img src="images/Icon-see-live.png" alt="live" />
-                </button>
-                <button type="button" class="modal-btn">
-                  See Source
-                  <img src="images/Icon-see-source.png" alt="Github" />
-                </button>
+            <div class="text-content">
+              <h2 class="tonic">${data[i].title}</h2>
+              <div class="profile">
+                <p class="canopy">Canopy</p>
+                <p class="back-end">
+                  <img class="counter" src="images/Counter.png" alt="counter" />
+                  Back End Dev
+                </p>
+                <p class="back-end">
+                  <img class="counter" src="images/Counter.png" alt="counter" />
+                  2015
+                </p>
               </div>
+              <p class="detail">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tools">
+                <li class="tool">html</li>
+                <li class="tool">css</li>
+                <li class="tool">javaScript</li>
+              </ul>
+              <button type="button" id="open-me" class='see-project see-project-${i}'>See Project</button>
             </div>
-          </div>
-        </div>
-      </div>
- `;
+          </div>`;
+}
+
+document.addEventListener('click', (click) => {
+  if (click.target.id === 'open-me') {
+    popUp.style.display = 'block';
+    modal.style.display = 'block';
+  }
 });
 
-const modalPopUp = () => {};
+for (let i = 0; i < data.length; i += 1) {
+  document.querySelector(`.see-project-${i}`).addEventListener('click', () => {
+    snapshoot1Desk.src = data[i].image;
+    modaltitle.innerHTML = data[i].title;
+  });
+}
```

Comment:
- [ ] [OPTIONAL] it would be better if you used the newer ES6 syntax to loop through the `data` object. it makes it more readable and concise.
something like this would be better:
```suggestion
data.forEach((item, i) => {
  const { image, title } = item;
  document.querySelector(`.see-project-${i}`).addEventListener('click', () => {
    snapshoot1Desk.src = image;
    modaltitle.innerHTML = title;
  });
})
```

Reply:
Thank you for the suggestion.

Reply:
no problem